### PR TITLE
Expose error codes

### PR DESCRIPTION
**Pull request by domerd**
Goes along with [this](https://github.com/tomlegkov/marine-python/pull/38) PR.

### CODE REVIEW

File: marine.h [hunk 1]
```diff
@@ -22,4 +22,9 @@ void destroy_marine(void);
 extern const unsigned int ETHERNET_ENCAP;
 extern const unsigned int WIFI_ENCAP;
 
+
+const int BadBPFErrorCode;
```
Comment (yehonatanz) [inline marine.h]: Shouldn't it be `extern`?

Comment (yehonatanz) [inline marine.h]: Also, consts are written in `ALL_CAPS`

**Review by yehonatanz — CHANGES_REQUESTED** [review]

File: marine.c [hunk 2]
```diff
@@ -89,6 +89,10 @@ static gboolean epan_auto_reset = TRUE;
 const unsigned int ETHERNET_ENCAP = 1;
 const unsigned int WIFI_ENCAP = 23;
 
+WS_DLL_PUBLIC_DEF const int BadBPFErrorCode = -1;
```
Comment (yehonatanz) [inline marine.c]: Fix the casing here too (you changed only in `marine.h`)

**Review by yehonatanz — CHANGES_REQUESTED** [review]

**Review by yehonatanz — APPROVED** [review]

File: marine.c [hunk 3]
```diff
@@ -89,6 +89,10 @@ static gboolean epan_auto_reset = TRUE;
 const unsigned int ETHERNET_ENCAP = 1;
 const unsigned int WIFI_ENCAP = 23;
 
+WS_DLL_PUBLIC_DEF const int BAD_BPF_ERROR_CODE = -1;
+WS_DLL_PUBLIC_DEF const int BAD_DISPLAY_FILTER_ERROR_CODE = -2;
+WS_DLL_PUBLIC_DEF const int INVALID_FIELD_ERROR_CODE = -3;§
```
Comment (tomlegkov) [inline marine.c]: Weird symbol

**Review by tomlegkov — CHANGES_REQUESTED** [review]

**Review by shaharyarn — APPROVED** [review]

**Comment by shaharyarn** [comment]
Closed in order to create a new PR.

**Pull request closed without merging**